src/style.css

```diff
@@ -1 +1,74 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --text-color: #3d4e61;
+  --border-color: #e1e7ed;
+}
+
+body {
```

Comment:
reset default body margin

Reply:
Still not fixed
<img width="853" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/07efe456-92bf-4dbe-9617-f43ce58bf20f">